<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-B78MGBJP" failures="3" tests="3" name="Pruebas de Funcionalidad de Login" time="329.126" errors="0" timestamp="2025-07-30T23:26:13 CLT">
  <testcase classname="com.automatizacion.proyecto.pruebas.PruebaLogin" name="testContrasenaInvalida" time="55.627">
    <failure type="java.lang.AssertionError" message="La página de login debería estar visible. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: La página de login debería estar visible. expected [true] but found [false]
at com.automatizacion.proyecto.pruebas.PruebaLogin.testContrasenaInvalida(PruebaLogin.java:81)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- testContrasenaInvalida -->
  <testcase classname="com.automatizacion.proyecto.pruebas.PruebaLogin" name="testInicioSesionExitoso" time="52.646">
    <failure type="java.lang.AssertionError" message="La página de login debería estar visible. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: La página de login debería estar visible. expected [true] but found [false]
at com.automatizacion.proyecto.pruebas.PruebaLogin.testInicioSesionExitoso(PruebaLogin.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- testInicioSesionExitoso -->
  <testcase classname="com.automatizacion.proyecto.pruebas.PruebaLogin" name="testUsuarioInvalido" time="180.022">
    <failure type="org.openqa.selenium.TimeoutException" message="java.util.concurrent.TimeoutException
Build info: version: &amp;apos;4.22.0&amp;apos;, revision: &amp;apos;c5f3146703*&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;21.0.8&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [5c3b5f7c453639733e720bbbd5255898, get {url=https://practice.expandtesting.com/login}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 138.0.7204.169, chrome: {chromedriverVersion: 138.0.7204.183 (e90faf484dd..., userDataDir: C:\Users\uman_\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:27335}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:27335/devtoo..., se:cdpVersion: 138.0.7204.169, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 5c3b5f7c453639733e720bbbd5255898">
      <![CDATA[org.openqa.selenium.TimeoutException: java.util.concurrent.TimeoutException
Build info: version: '4.22.0', revision: 'c5f3146703*'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.8'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [5c3b5f7c453639733e720bbbd5255898, get {url=https://practice.expandtesting.com/login}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 138.0.7204.169, chrome: {chromedriverVersion: 138.0.7204.183 (e90faf484dd..., userDataDir: C:\Users\uman_\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:27335}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:27335/devtoo..., se:cdpVersion: 138.0.7204.169, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 5c3b5f7c453639733e720bbbd5255898
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:399)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute(JdkHttpClient.java:355)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:187)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:300)
at com.automatizacion.proyecto.paginas.PaginaLogin.navegarA(PaginaLogin.java:45)
at com.automatizacion.proyecto.pruebas.PruebaLogin.testUsuarioInvalido(PruebaLogin.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.util.concurrent.TimeoutException
at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1960)
at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2095)
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:382)
... 30 more
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- testUsuarioInvalido -->
</testsuite> <!-- Pruebas de Funcionalidad de Login -->
